[게시글 생성/조회/수정/삭제] 올라프 미션 제출합니다.

school0bhy opened the pull request:
처음 스프링을 써봐서 많이 부족합니다.
많은 지적 부탁드립니다!

school0bhy committed:
docs : README.md 작성

school0bhy committed:
feat: 게시글 작성 페이지 이동 구현

* index.html, article-edit.html 파일을 templates 폴더로 이동함

* AriticleController.java: /writing GET요청 처리

Squashed commit of the following:

commit 7fa33210c68665351585da9f8fbc4e2117248a2e
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Tue Jul 9 17:26:55 2019 +0900

    docs: TODO 추가

commit d8e14dbdd71112723278e2e312ec46582a131d00
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Tue Jul 9 17:26:17 2019 +0900

    feat: 게시글 작성 페이지 이동

school0bhy committed:
feat: 게시글 작성 및 추가 기능 구현

school0bhy committed:
feat: 게시글 목록 조회

commit 9361045bbb8cfc607918c5ed4bda4ceee9aece6f
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Wed Jul 10 18:01:03 2019 +0900

    feat: 게시글 목록 조회

school0bhy committed:
feat: 게시글 조회 기능

Squashed commit of the following:

commit 513b0f9f8dcf8cd2a2810db7d8e82c3efd13b317
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Thu Jul 11 08:42:08 2019 +0900

    feat: 게시글 조회 기능 구현

commit 55b1b0ea2760d83c12b1cc060e2fa0d03e9d8297
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Thu Jul 11 08:29:05 2019 +0900

    chore: 절대경로로 변경

    * 외부 리소스 접근 경로 상대경로에서 절대경로로 변경

school0bhy committed:
feat: 게시글 수정 기능 구현

school0bhy committed:
feat: 게시글 삭제 기능 구현

Squashed commit of the following:

commit 44fa3af15a2b1b6da8b8229f06a26c315cfa493c
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 14:11:07 2019 +0900

    docs: TODO.md 추가

commit b01f06e590c2b8b42a40c8da36adb01705bfbf7f
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 14:01:39 2019 +0900

    refactor: html js부분 중복 제거

commit 51101469f093fd305653c6ed63d5be8aaa69ad24
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 09:52:07 2019 +0900

    fix: article-edit.html 나가기 버튼 오류 수정

commit 23ba3a40a3c427248c4ee8f1bdbf16a1119d1798
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 09:49:11 2019 +0900

    feat: 게시글 삭제 시 redirect

commit 70d5a2e9232f2baea54bbe3ac554696b89ad9149
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 09:29:53 2019 +0900

    feat: 게시글 삭제 후 남은 게시글 보여주기

commit 184ddc4acf42506d2f46d48eb4e79b4dae6173ce
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 09:21:21 2019 +0900

    refactor: 들여쓰기

commit 4e8b30e3dfa72ed4babfba2f4e3ffd9b571f8787
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 09:16:19 2019 +0900

    feat: 게시글 삭제 기능 구현

commit 5111c6e2d26adec5799a4546a3100a0e3f874835
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 01:43:53 2019 +0900

    refactor: Article 클래스 id 필드 추가

commit 2e71b5f6a5010bbe5cecca9a4f26616f53788450
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Fri Jul 12 01:42:44 2019 +0900

    chore: html 중복 제거

commit b88d426c35f40c12d1e069b9480789d0c7937563
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Thu Jul 11 16:54:29 2019 +0900

    refactor: UI 버그 수정

    * app.css, index.html 수정

commit 7c80edaf0f4452a5f610487086c466ce0da8292a
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Thu Jul 11 16:49:31 2019 +0900

    docs: 게시글 삭제 기능 TODO 작성

school0bhy committed:
refactor: InvalidArticleException 로 Article 예외 처리

jihan805 reviewed (changes requested):
안녕하세요 ~ 구현 👍  
몇가지 피드백 남겼으니 확인해주세요 :)

school0bhy committed:
refactor : 피드백 반영

Squashed commit of the following:

commit 3502b6bbf150aeae8ca67faba9651d1deabf84f1
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:23:34 2019 +0900

    refactor: field injection에서 constructor injection으로 수정

commit 9cdcde68f444456e08d71853fc07a0861536048e
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:14:39 2019 +0900

    refactor: ArticleController 클래스 PathVariable 자료형 int로 통일

commit 72a7c41d5e089876c4a126c44bf2f369c6602bf3
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:11:47 2019 +0900

    refactor: 테스트 코드 중복 제거

commit 71766eb20f8af1c57ad2f6453da7d0cedaec8b54
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 12:34:41 2019 +0900

    refactor: Article 클래스에서 게시물 수정 메서드(update) 구현

commit ff7daec7c63371a5f0901c9dedd25924dd998cee
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 12:33:23 2019 +0900

    docs: 피드백 관련 TODO.md 작성

school0bhy committed:
Squashed commit of the following:

commit b3fb54bf50e96576f2e6c2ad68da298c291d0a2e
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 15:03:43 2019 +0900

    refactor: 수정한 게시글 내용 확인 테스트를 추가

commit da8e30ca3f936c90c1a0d49f9d84cbd64142d42b
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:53:12 2019 +0900

    refactor: test 코드 중복 제거

commit 5b1abb3a6ca22e256256ebbc475ac80437d9166e
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:48:01 2019 +0900

    refactor: html 중복제거

commit 805f23d18afe93a0b327a8abbb9745ad0ff551ae
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:44:51 2019 +0900

    refactor: ArticleRepository 클래스의 find 메서드 findById 로 메서드명 변경

commit b4ebb395c7f166f685bcb819e43b1222de5c1ca5
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:42:20 2019 +0900

    refactor: ArticleDto에 toArticle 메서드를 구현

commit 8ad5c7f015098e682556ad578626d6b4a0795532
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:41:29 2019 +0900

    refactor: 테스트 코드 수정

    * ArticleNotFoundException 적용

commit 7eb3c5f3dbfa8aba6b30b80c337ffa104e96b36f
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:37:21 2019 +0900

    refactor: ArticleNotFoundException 추가

commit 0bd73b3de30c220e2e9b869f9e525d0da5070759
Author: school0bhy <house1217@ajou.ac.kr>
Date:   Sun Jul 14 14:35:48 2019 +0900

    refactor: build.gradle 컨벤션, 테스트 실패 부분 통과하도록 수정

jihan805 reviewed (approved):
안녕하세요~! 피드백 반영 잘하셨어요 💯 
머지 할게요 :)
